src/definitions/behaviors/form.js

```diff
@@ -1993,6 +1993,9 @@ $.fn.form.settings = {
 
       // allow dashes in card
       cardNumber = cardNumber.replace(/[\-]/g, '');
+      
+      // allow spaces in card
+      cardNumber = cardNumber.replace(/\s/g, '');
```

Comment:
Nit) I think these regex can be consolidated.

```suggestion
      // allow dashes and spaces in card
      cardNumber = cardNumber.replace(/[\s\-]/g, '');
```